generated/niscope/unit_tests/mock_helper.py

```diff
@@ -273,17 +273,11 @@ def niScope_Fetch(self, vi, channel_list, timeout, num_samples, wfm, wfm_info):
         if self._defaults['Fetch']['Wfm'] is None:
             raise MockFunctionCallError("niScope_Fetch", param='Wfm')
         a = self._defaults['Fetch']['Wfm']
-        import sys
-        if sys.version_info.major > 2 and type(a) is str:
-            a = a.encode('ascii')
         for i in range(min(len(wfm), len(a))):
             wfm[i] = a[i]
         if self._defaults['Fetch']['wfmInfo'] is None:
             raise MockFunctionCallError("niScope_Fetch", param='wfmInfo')
         a = self._defaults['Fetch']['wfmInfo']
-        import sys
-        if sys.version_info.major > 2 and type(a) is str:
-            a = a.encode('ascii')
```

Comment:
I wonder if the six library can take care of this for us, where we make a single function call and it does the right thing based on Python version without us needing to add conditional logic. Worth looking into (can go into separate PR).

Reply:
I'd rather make it a separate PR